lib/config/config-file.js

```diff
@@ -329,11 +329,7 @@ function applyExtends(config, filePath) {
             debug("Loading " + parentPath);
             return ConfigOps.merge(load(parentPath), previousValue);
         } catch (e) {
-            // If the file referenced by `extends` failed to load, add the path to the
-            // configuration file that referenced it to the error message so the user is
-            // able to see where it was referenced from, then re-throw
-            e.message += "\nReferenced from: " + filePath;
-            throw e;
+            return {}; // suppress error for unsupported extensions
```

Comment:
That's my thinking.

An alternative approach might be to only attempt to `load(parentPath)` when it points to one of a few `airbnb` configurations.